src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -11,36 +14,86 @@ export default function PopupWithForm({ name, onSwitch }) {
           onClick={() => onSwitch("")}
         />
         {name === "signup" && <FormSignUp onSwitch={onSwitch} />}
-        {name === "signin" && <FormSignIn onSwitch={onSwitch} />}
-        {name === "success" && <FormSucess onSwitch={onSwitch}/>}
+        {name === "signin" && <FormSignIn onSwitch={onSwitch} closeModal={() => onSwitch("")}/>}
+        {name === "success" && <FormSucess onSwitch={onSwitch} />}
       </div>
     </div>
   );
 }
 
 function FormSignUp({ onSwitch }) {
```

Comment:
**Might be better**


it's better to put different components to different files to structure the code